.gitignore

```diff
@@ -26,3 +26,4 @@ core/CMakeCache.txt
 **/CMakeFiles
 .envrc
 .vscode
+node_modules
```

Comment:
why? Web Console should is not longer part of the project. Perhaps it is easier to remove local directory ?

Reply:
Must be a leftover . Fixed in 2f5e8c44 .